src/VisualStudio/Core/Def/Implementation/Preview/ReferenceChange.cs

```diff
@@ -101,7 +101,7 @@ public sealed override int GetTipText(out VSTREETOOLTIPTYPE eTipType, out string
 
         public sealed override int OnRequestSource(object pIUnknownTextView)
         {
-            if (pIUnknownTextView != null && Children.Changes != null && Children.Changes.Length > 0)
+            if (pIUnknownTextView != null && Children?.Changes != null && Children.Changes.Length > 0)
```

Comment:
Why/when does this happen?

Reply:
This happens when a ProjectReference is added and the user is looking at the preview and clicks on the Project reference node.

![image](https://user-images.githubusercontent.com/611219/45905815-9d88a900-bda6-11e8-9884-aeea5a4113d3.png)

Reply:
Great.  can you add a comment to the code?  Thanks!